requests_operating_unit/models/request.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2015-TODAY Eficent
+# - Jordi Ballester Alomar
+# Copyright 2015-TODAY Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models
+
+
+class OperatingUnit(models.Model):
```

Comment:
We don't need this model. We just inherit module, operating_unit in __manifest__.py and we are good.